src/ledger/shred.zig

```diff
@@ -799,6 +802,7 @@ pub const CommonHeader = struct {
     slot: Slot,
     index: u32,
     version: u16,
+    // Question: Is this also set when it is a data shred
```

Comment:
Yes, each erasure set includes data shreds and coding shreds. fec_set_index represents the erasure set within a slot. "fec set" and "erasure set" are synonyms as far as I'm concerned.

I think the fec_set_index might actually be based on a data shred's index? Not 100% sure about this though